src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/CounterGroup.cs

```diff
@@ -47,17 +48,27 @@ private void OnEventSourceCommand(object? sender, EventCommandEventArgs e)
 
                 if (e.Arguments.TryGetValue("EventCounterIntervalSec", out string? valueStr) && float.TryParse(valueStr, out float value))
                 {
-                    lock (s_counterGroupLock)      // Lock the CounterGroup
+                    lock (s_counterGroupLock)
                     {
-                        EnableTimer(value);
+                        if (s_refCount == 0)
+                        {
+                            EnableTimer(value);
+                        }
+
+                        ++s_refCount;
                     }
                 }
             }
             else if (e.Command == EventCommand.Disable)
             {
                 lock (s_counterGroupLock)
                 {
-                    DisableTimer();
+                    if (s_refCount == 1)
+                    {
+                        DisableTimer();
```

Comment:
It looks like EnableTimer might spin up a polling thread.  Does it matter that DisableTimer doesn't tear that thread down?